packages/flutter/lib/src/widgets/table.dart

```diff
@@ -201,7 +201,9 @@ class Table extends RenderObjectWidget {
   /// determine the intrinsic size of the column.
   ///
   /// The keys of this map (column indexes) are zero-based.
-  final Map<int, TableColumnWidth> columnWidths;
+  ///
+  /// If this is set to null, then an empty map is assumed.
+  final Map<int, TableColumnWidth>/*?*/ columnWidths;
```

Comment:
Why is this still specified as an annotation?

Reply:
because this file is still opted-out (in widgets)

Reply:
Ah, that makes sense! Thanks.

Reply:
we haven't migrated widgets/ yet. This is just making sure we make the right call when we get to it.